src/commonmark/en/content/developer/web-api.md

```diff
@@ -5969,6 +5969,90 @@ from the following
 
     GET api/dataApprovals/categoryOptionCombos?wf=rIUL3hYOjJc&pe=201801&ou=YuQRtpLP10I
 
+### Bulk get approval status
+
+To get a list of multiple approval statuses, you can issue a GET request similar to this:
+
+    GET http://server.com/api/dataApprovals/multiple?wf=rIUL3hYOjJc&pe=201801,201802&ou=YuQRtpLP10I
+
+The parameters wf, pe, ou, and aoc are the same as for getting a single approval status, except that you can provide a comma-separated list of one or more values for each parameter.
+
+This will give you a response containing a list of approval parameters and statuses, something like this:
+
+    [
+      {
+          aoc: "HllvX50cXC0",
+          pe: "201801",
+          level: "KaTJLhGmU95",
+          ou: "YuQRtpLP10I",
+          permissions: {
+              mayApprove: false,
+              mayUnapprove: true,
+              mayAccept: true,
+              mayUnaccept: false,
+              mayReadData: true
+          },
+          state: "APPROVED_HERE",
+          wf: "rIUL3hYOjJc"
+      },
+      {
+          aoc: "HllvX50cXC0",
+          pe: "201802",
+          ou: "YuQRtpLP10I",
+          permissions: {
+              mayApprove: true,
+              mayUnapprove: false,
+              mayAccept: false,
+              mayUnaccept: false,
+              mayReadData: true
+          },
+          state: "UNAPPROVED_READY",
+          wf: "rIUL3hYOjJc"
+      }
+    ]
+
+The returned parameters are:
+
+<table>
```

Comment:
It would be great if we could use a GFM table here instead! 
(In general, I want people to use GFM (pipe-based) tables wherever possible as I think they are cleaner, and they really help with translation).
e.g.:

**Bulk data approval returned parameters**
| Return Parameter | Description |
| :- | :--- |
| aoc | Attribute option combination identifier |
| pe | Period identifier |
| ou | Organisation Unit identifier |
| permissions | The permissions: 'mayApprove', 'mayUnapprove', 'mayAccept', 'mayUnaccept', and 'mayReadData' (same definitions as for get single approval status.) |
| state | One of the data approval states (same as for get single approval status.) |
| wf | Data approval workflow identifier |

Reply:
I was just copying what was nearby. But if mixing table types doesn't matter in a section, I agree that we should be using markdown tables for new content (and maybe converting old HTML ones too.)

Reply:
Thanks Jim. I think mixing is fine! I think the more GFM tables we have, the better.

I hope to convert the old ones if/when time permits... (It can be automated a little bit, but not very easily, and usually needs some manual tweaking)